Jenkinsfile

```diff
@@ -29,63 +29,6 @@ pipeline {
       }
     }
 
-    stage('Build Docker Images'){
-      steps {
-        withCredentials([string(credentialsId: 'sidekiq-enterprise-license', variable: 'BUNDLE_ENTERPRISE__CONTRIBSYS__COM')]) {
-          sh 'env=$RAILS_ENV make build'
-        }
-      }
-    }
-
-    stage('Setup Testing parallel DBs') {
-      steps {
-        sh 'env=$RAILS_ENV make spec_parallel_setup'
-      }
-    }
-
-    stage('Lint Changed Files') {
-      when { changeRequest() }
-      steps {
-        script {
-          files_to_lint = ''
-          try {
-            files_to_lint = getGithubChangedFiles('vets-api', env.CHANGE_ID.toInteger(),
-                              change_types: ['modified', 'added']).join(' ')
-          } catch(IOException e) {
-            echo "WARNING: Unable to fetch changed PR files from Github!"
-            echo "${e}"           
-          }
-        }
-        sh """env=$RAILS_ENV make files='${files_to_lint}' lint"""
-      }
-    }
-
-    stage('Lint All Files') {
-      when { branch 'master' }
-      steps {
-        sh 'env=$RAILS_ENV make lint'
-      }
-    }
-
-    stage('Security Scan') {
-      steps {
-        sh 'env=$RAILS_ENV make security'
```

Comment:
Should we remove the associated lint/security rake files? Or should we use them in GHA instead?

Reply:
Interesting question. Is there a reason why they weren't used within GHA? I think I'm indifferent on this particular point, removing them since they aren't used would be the quickest solution.

Reply:
I think I'd prefer to leave them in though for now. They could be used within GHA.